### PR TITLE
feat: welcome page

### PR DESCRIPTION
**Pull request by huaxiabuluo**
- feat: welcome page
- update: golang@1.18
- fix: login logic

### CODE REVIEW

File: app/pages/Welcome/index.tsx [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    tip: 'doc.schemaIntro',
+    startLink: '/schema',
+    docLink: 'https://docs.nebula-graph.com.cn/2.0.1/3.ngql-guide/9.space-statements/1.use/',
+  },
+  {
+    icon: 'icon-studio-nav-import',
+    title: 'import.importData',
+    tip: 'doc.importIntro',
+    startLink: '/import/files',
+    docLink: 'https://docs.nebula-graph.com.cn/2.0.1/3.ngql-guide/9.space-statements/1.use/',
```
Comment (hetao92) [inline app/pages/Welcome/index.tsx]: i18n doc

File: app/pages/Welcome/index.tsx [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+                  <div className={styles.tagWrapper}>
+                    {dataset.tags.map((tag) => (
+                      <Tag key={tag}>{tag}</Tag>
+                    ))}
+                  </div>
+                  <div className={styles.infoWrapper}>
+                    <span>
+                      Edge: <strong>{dataset.detail.edgeCount}</strong>
+                    </span>
+                    <span>Vertex: {dataset.detail.vertexCount}</span>
```
Comment (hetao92) [inline app/pages/Welcome/index.tsx]: edge & vertex has i18n in locale

File: app/config/locale/zh-CN.json [hunk 3]
```diff
@@ -293,5 +293,25 @@
...
+    "doc": "文档",
+    "guide": "新手引导",
+    "quickStart": "开始",
+    "quickStartDesc": "文档",
+    "explorerGraph": "探索图数据",
+    "explorerGraphTip": "It is used with the Nebula Graph core to visualize interaction with graph data.",
+    "visualQuery": "可视化查询",
+    "visualQueryTip": "It is used with the Nebula Graph core to visualize interaction with graph data.",
+    "workflow": "工作流",
+    "workflowTip": "It is used with the Nebula Graph core to visualize interaction with graph data.",
```
Comment (hetao92) [inline app/config/locale/zh-CN.json:316]: update / delete

**Review by mizy — APPROVED** [review]
LGTM

**Pull request merged**